src/components/Table/TableBody/TableBodyCell/TableBodyCell.tsx

```diff
@@ -24,6 +24,14 @@ interface TableCellProps {
    * Reduces padding inside table cells.
    */
   isCompact?: boolean;
+  /**
+   * Will stick to left side of table during horizontal scroll.
+   */
+  isColumnStickyLeft?: boolean;
+  /**
+   * Will stick to right side of table during horizontal scroll.
+   */
+  isColumnStickyRight?: boolean;
```

Comment:
instead of having two separate booleans, recommend changing this to a single prop that accepts `left | right`. This prevents accidentally having both `isColumnStickyLeft` and `isColumnStickyRight` at the same time.